src/std_extensions/collections.rs

```diff
@@ -47,6 +52,13 @@ impl CustomConst for ListValue {
         SmolStr::new_inline("list")
     }
 
+    fn typ(&self) -> CustomType {
+        let list_type_def = EXTENSION.get_type(&LIST_TYPENAME).unwrap();
```

Comment:
You could use `list_type` (toplevel in collections.rs) here

Reply:
No, I mean isn't this whole method just `list_type(self.0.clone())` ?

Reply:
No, `list_type` returns a `Type` and we need a `CustomType`.

Reply:
Ah ok yes you are right, sorry for noise!